mirrord/intproxy/src/proxies/incoming.rs

```diff
@@ -270,6 +274,40 @@ impl IncomingProxy {
                     interceptor.send(req).await;
                 }
             }
+            DaemonTcp::HttpRequestChunked(req) => {
+                tracing::error!("{:?}", req);
+                match req {
+                    ChunkedRequest::Start(req) => {
+                        let (tx, rx) = mpsc::channel::<ChunkedFrame>(12);
+                        let http_req = HttpRequest {
+                            internal_request: InternalHttpRequest {
+                                method: req.internal_request.method,
+                                uri: req.internal_request.uri,
+                                headers: req.internal_request.headers,
+                                version: req.internal_request.version,
+                                body: (),
+                            },
+                            connection_id: req.connection_id,
+                            request_id: req.request_id,
+                            port: req.port,
+                        };
+                        let http_stream = ReceiverStreamReplay::new(rx);
+
+                        // TODO: pass frames to tx
+                        for frame in req.internal_request.body.iter().cloned() {
```

Comment:
```suggestion
                        for frame in req.internal_request.body {
```
No need to clone, right?